keygen/populate_keys.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+import click
```

Comment:
`argparse` is really cumbersome to work with, but a lot more portable. if you used `argparse` instead of `click`, you wouldn't need a virtualenv for this script

Reply:
great point.  i'll consider refactoring to use `argparse`, just anticipating this repo will grow additional dependencies soon.